hermes-common/src/main/java/pl/allegro/tech/hermes/common/metric/HermesMetrics.java

```diff
@@ -252,13 +255,6 @@ public Timer consumersWorkloadRebalanceDurationTimer(String kafkaCluster) {
         return metricRegistry.timer(pathCompiler.compile(Timers.CONSUMER_WORKLOAD_REBALANCE_DURATION, pathContext));
     }
 
-    public void reportConsumersWorkloadStats(String kafkaCluster, int missingResources, int deletedAssignmentsCount, int createdAssignmentsCount) {
-        PathContext pathContext = pathContext().withKafkaCluster(kafkaCluster).build();
-        metricRegistry.histogram(pathCompiler.compile(Histograms.CONSUMERS_WORKLOAD_SELECTIVE_MISSING_RESOURCES, pathContext)).update(missingResources);
```

Comment:
Should we delete the constants? If the gauges use the same names then we should make sure they don't get reused.